docs/basics/schema.md

```diff
@@ -105,7 +105,8 @@ Use `dataType(Class)` to define the data type of a property key.
 JanusGraph will enforce that all values associated with the key have the
 configured data type and thereby ensures that data added to the graph is
 valid. For instance, one can define that the `name` key has a String
-data type.
+data type. Note that primitive types are not supported. Use the corresponding
+object type, e.g. Integer instead of int.
```

Comment:
Please highlight _Integer_  and _int_ as code by using ``: `Integer` and `int`.

Also, are those really called _object types_? I just searched for a term for these and found either _wrapper classes_ or _reference types_.

Reply:
Edited. I agree wrapper class or reference type sounds more correct. I was using 'object type' simply because the code said so: https://github.com/JanusGraph/janusgraph/blob/d0e5eff1de4e0dcdc95b6292ebe64a3baa357646/janusgraph-core/src/main/java/org/janusgraph/graphdb/types/StandardPropertyKeyMaker.java#L103-L106

Reply:
Oh, I didn't compare the two messages. Would be good to have this consistent between docs and the error message so it would be good if you could also update the error message.

Reply:
Sure, done